include/tester/db/Database.hpp

```diff
@@ -76,8 +48,9 @@ class Database
 
     bsoncxx::stdx::optional<std::string> getField(std::string collectionName, std::string id, std::string field_name)
     {
-        mongocxx::collection collection = _database[collectionName];
-        auto result = collection.find_one(bsoncxx::builder::stream::document{} << "_id" << bsoncxx::oid{id} << bsoncxx::builder::stream::finalize);
+        mongocxx::collection collection = _database[std::move(collectionName)];
+        auto result = collection.find_one(bsoncxx::builder::stream::document{} << "_id" 
+            << bsoncxx::oid{id} << bsoncxx::builder::stream::finalize);
```

Comment:
`id` needs to be passed with `std::move` here

Reply:
done